code/game/machinery/cryopod.dm

```diff
@@ -291,9 +291,9 @@
 //Lifted from Unity stasis.dm and refactored.
 /obj/machinery/cryopod/Process()
 	if(occupant)
-		if(applies_stasis && iscarbon(occupant) && (world.time > time_entered + 20 SECONDS))
-			var/mob/living/carbon/C = occupant
-			C.SetStasis(2)
+		if(applies_stasis && (world.time > time_entered + 20 SECONDS))
+			var/mob/living/patient = occupant
```

Comment:
do you need to cast here if occupant is already /living? Maybe a typecheck if nothing prevents non living form entering

Reply:
> do you need to cast here if occupant is already /living? Maybe a typecheck if nothing prevents non living form entering

That might be more appropriate in set_occupant, and then we can just always assume occupant is either living or null.